src/multipole_header.F90

```diff
@@ -0,0 +1,152 @@
+module multipole_header
+
+  implicit none
+
+  !========================================================================
+  ! Multipole related constants
+
+  ! Formalisms
+  integer, parameter :: FORM_MLBW = 2, &
+                        FORM_RM   = 3, &
+                        FORM_LRM  = 7
+
+  ! Constants that determine which value to access
+  integer, parameter :: MP_EA = 1       ! Pole
+
+  ! Reich-Moore indices
+  integer, parameter :: RM_RT = 2, &    ! Residue total
+                        RM_RA = 3, &    ! Residue absorption
+                        RM_RF = 4       ! Residue fission
+
+  ! Multi-level Breit Wigner indices
+  integer, parameter :: MLBW_RT = 2, &  ! Residue total
+                        MLBW_RX = 3, &  ! Residue compettitive
+                        MLBW_RA = 4, &  ! Residue absorption
+                        MLBW_RF = 5     ! Residue fission
+
+  ! Polynomial fit indices
+  integer, parameter :: FIT_T = 1, &    ! Total
+                        FIT_A = 2, &    ! Absorption
+                        FIT_F = 3       ! Fission
+
+  ! Value of 'true' when checking if nuclide is fissionable
+  integer, parameter :: MP_FISS = 1
+
+  ! These variables store the maximum value from every nuclide in order
+  ! to preallocate some arrays to improve performance.
+  integer :: max_poly  ! Maximum number of polynomials we expect
+  integer :: max_poles ! Maximum number of poles in the problem for allocation
+  integer :: max_L     ! Maximum L value for allocation
+
+!===============================================================================
+! MULTIPOLE contains all the components needed for the windowed multipole
+! temperature dependent cross section libraries for the resolved resonance
+! region.
+!===============================================================================
+
+  type MultipoleArray
+
+    !=========================================================================
+    ! Isotope Properties
+    logical                 :: fissionable = .false.  ! Is this isotope fissionable?
+    integer                 :: length                 ! Number of poles
+    integer, allocatable    :: l_value(:)             ! The l index of the pole
+    real(8), allocatable    :: pseudo_k0RS(:)         ! pseudo_k0RS for each of l
```

Comment:
Comment is not terribly helpful. Can you elaborate in a comment somewhere what exactly pseudo_k0RS is?